client/app/components/blueprint-details-modal/order-list.directive.js

```diff
@@ -38,20 +38,18 @@
         list: '='
       },
       templateUrl: 'app/components/blueprint-details-modal/order-list.directive.html',
-      controller: ['$scope', '$filter', OrderListController],
-      controllerAs: 'vm'
+      controller: OrderListController
     };
 
     return directive;
   }
 
   function OrderListController($scope, $filter) {
-    var vm = this;
-    vm.listDisabled = false;
+    $scope.listDisabled = false;
```

Comment:
Hi, @chriskacerguis , just IM'ing with @serenamarie125 , she is creating eslint JIRA stories to cover this work.